x-pack/plugins/stack_connectors/server/connector_types/gemini/gemini.test.ts

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { GeminiConnector } from './gemini';
+import { ServiceParams } from '@kbn/actions-plugin/server';
+import { Config, Secrets, RunActionParams } from '../../../common/gemini/types';
+import { ActionsConfigurationUtilities } from '@kbn/actions-plugin/server/actions_config';
+import { Logger } from '@kbn/logging';
+import type { Services } from '@kbn/actions-plugin/server/types';
+import {
+  KibanaRequest,
+  SavedObjectsClientContract,
+  ElasticsearchClient,
+  KibanaRequestEvents,
+  IKibanaSocket,
+} from '@kbn/core/server';
+import { ConnectorTokenClient } from '@kbn/actions-plugin/server/lib/connector_token_client';
+import type { Headers } from 'packages/core/http/core-http-server/src/router/headers';
```

Comment:
I don't think you need to define this type, I'm seeing an error on the import

Reply:
cc: @shivankawasthi11

Reply:
@stephmilovic The reason that I had to import this type as Header is a required parameter for "KibanaRequest" class which in turn is a required parameter for ServiceParams interface. ServiceParams is a required parameter to create a GeminiConnector instance